tensorflow/compiler/xla/tests/convolution_test.cc

```diff
@@ -1946,7 +1948,9 @@ XLA_TEST_F(ConvolutionTest, ConvolveF32BackwardInputGroupedConvolution) {
 
 class ConvolutionHloTest : public HloTestBase {};
 
-XLA_TEST_F(ConvolutionHloTest, ConvolveF64Forward) {
+// double datatype is not yet supported in ROCm
+XLA_TEST_F(ConvolutionHloTest,
+           DISABLED_ON_GPU_ROCM(DISABLED_ON_CPU(ConvolveF64Forward))) {
```

Comment:
Why did you add DISABLED_ON_CPU here and below?

Reply:
thanks for catching this. let me fix it.